css/index.css

```diff
@@ -20,12 +20,17 @@ header {
   border-bottom: 1px solid #DDDDDD;
 }
 
-nav {
+header nav {
```

Comment:
The `<section>` is changed to a `<nav>` to improve semantics, which improves a11y and the selector needed to be tightened a bit